src/Context/Filter Context/filter-context.js

```diff
@@ -0,0 +1,26 @@
+import { createContext, useContext, useReducer } from 'react';
+import { filterReducer } from '../../Reducer';
+
+const FilterContext = createContext({
+	price: 5000,
+	category: [],
+	rating: 1,
+	sortBy: null,
+	outOfStock: false,
+});
```

Comment:
Just a small suggestion, you can also move the initial state in the util file.

Reply:
Since the code is only of 5 lines, I think it is okay for. Thank you for the suggestion tho @Kevin-Solomon 👍